Topics/User_Experience/Button_Design.md

```diff
@@ -0,0 +1,77 @@
+# **Button Design**
+
+## **Introduction**
+When creating software, 99.9% of the time, we will need at least one button for each page. Buttons are one of the most common interactions between users and software. However, many software developers underestimate how important buttons are in terms of user experience. A poorly designed button can be confusing, misleading, or even fraustrating enough so that users give up on using the softare. On this page, we will introduce multiple ways that a button can be designed, and their implication behind each design.
+
+## **Button Accessibility (with implementation examples using React Native)**
+We will be using the libraries below in React Native for implementations of our button design examples.
+
+```
+import { AntDesign } from '@expo/vector-icons'; 
+import { Button } from 'react-native-elements';
+import Icon from 'react-native-vector-icons/FontAwesome';
+```
+
+### **Clickable vs Non-Clickable**
+While some buttons are design to be clicked, some are there to deliver the message "this is not an option" to users. Buttons play a significant role in user visibility since software users should be able to tell what is an option for them and what is not in a straight-forward way. 
+
+**Code Demo:**
+This is a regular button:
+```
+<Button
+  title="Button"
+/>
+```
+
+And this is a disabled button:
+```
+<Button
+  title="Disabled button"
+  disabled
+/>
+```
+More styling can be done by `disabledStyle` and `disabledTitleStyle`. 
+
+And similarly to disabled button, loading button can also be suitable under certain scenario. e.g. click to expand a paragrah. This is a loading button:
+```
+<Button
+  title="Disabled button"
```

Comment:
I am a bit confused by the title. Since this is a loading button, shouldn't the title be "Loading button" or something similar? Feel feel to correct me if I misunderstand something.

Reply:
Thank you for pointing this out! I was meant to introduce to difference between clickable buttons and not clickable buttons here. In some cases, especially the developing phase of the app when some functionalities are not fully implemented. Then, it would make sense to include a disabled button that tells the users this function is currently under development, but it will be there in the future. Loading button is also a design option for such use case. Let me try describing a bit more to avoid the confusion!

Reply:
Got it! Thanks for the explanation!